### PR TITLE
[#1261] Implement CCTP manual route for solana

### LINKED ISSUE #1261: [Enhancement] Add support for 2-tx Solana USDC transfers via CCTP
## Describe the issue in detail

 - Users should be able to transfer USDC into and out of Solana via CCTP, i.e. they should receive native Solana USDC when transferring into Solana
 - Users should be able to perform USDC transfers via 2 transactions (manually redeeming on the destination side)
 - 2-tx transaction USDC transfers should use CCTP directly (similar to how manual USDC transfers work between currently connected CCTP chains)
 
 Target completion date 12/1.

### PR DESCRIPTION
**Pull request by M-Picco**

### CODE REVIEW

File: wormhole-connect/src/routes/cctpManual/utils/chains.ts [hunk 1]
```diff
@@ -14,8 +14,23 @@ export const CCTPManual_CHAINS: ChainName[] = [
...
+  avalanche: 1,
+  fuji: 1,
+  goerli: 0,
+  base: 6,
+  optimism: 2,
+  arbitrum: 3,
+  optimismgoerli: 2,
+  arbitrumgoerli: 3,
+  basegoerli: 6,
+  solana: 5,
```
Comment (barnjamin) [inline wormhole-connect/src/routes/cctpManual/utils/chains.ts:31]: polygon?

Comment (kev1n-peters) [reply wormhole-connect/src/routes/cctpManual/utils/chains.ts:31]: good catch

File: sdk/src/contexts/solana/context.ts [hunk 2]
```diff
@@ -404,7 +404,7 @@ export class SolanaContext<
     );
 
     transaction.feePayer = payerPublicKey;
-    await addComputeBudget(this.connection!, transaction, [NATIVE_MINT]);
```
Comment (kev1n-peters) [inline sdk/src/contexts/solana/context.ts:407]: We don't need to pass the accounts anymore - `addComputeBudget` finds them for us

File: wormhole-connect/src/routes/cctpManual/cctpManual.ts [hunk 3]
```diff
@@ -281,53 +250,31 @@ export class CCTPManualRoute extends BaseRoute {
...
   ): Promise<string> {
-    const fromChainId = config.wh.toChainId(sendingChain);
-    const fromChainName = config.wh.toChainName(sendingChain);
-    const decimals = getTokenDecimals(fromChainId, token);
-    const parsedAmt = utils.parseUnits(amount, decimals);
-
-    // only works on EVM
-    if (!isEvmChain(sendingChain)) {
-      throw new Error('No support for non EVM cctp currently');
+    let recipientAccount = recipientAddress;
```
Comment (kev1n-peters) [inline wormhole-connect/src/routes/cctpManual/cctpManual.ts:253]: The recipient address is the associated token account

**Review by artursapek — APPROVED** [review]

**Pull request merged**